src/Twig/Extension/SeoExtension.php

```
@@ -264,4 +268,17 @@ private function normalize($string)
     {
         return htmlentities(strip_tags($string), ENT_COMPAT, $this->encoding);
     }
+
+    private function renderAttributes(AttributeBag $attributes, \Twig_Environment $environment = null): string
+    {
+        if (null === $environment) {
+            return '';
+        }
+
+        try {
```

What do you tink. Should I use this try catch for the template rendering? Or just let the exception bubble up?

---

I think you should let the exception bubble up, let people know where they failed.